### ISSUE TITLE
Calling createPaymentMethod causes app to crash and close.

### ISSUE THREAD

**Issue by ryanwaite28** [stripe/stripe-react-native#1240]
**Describe the bug**
Currently using React Native with Expo. Using @stripe/stripe-react-native version 0.22.1 (latest as of now on 12/10/2022).
A view render's the **CardField** component from the SDK and when trying to call createPaymentMethod,
expo shuts down. The error i get:

NSInvalidArgumentException: -[NSNull _fastCStringContents:]: unrecognized selector sent to instance 0x1fe57a740

NOTE: this only happens on iOS. i am using a physical iPhone 14 Pro running iOS 16

**To Reproduce**
1. Fill out CardField from sdk
2. Call createPaymentMethod

**Expected behavior**
It works fine on physical Android

**Screenshots**
![Screenshot 2022-12-10 at 2 27 09 PM](https://user-images.githubusercontent.com/12706937/206872343-c58a0482-37da-415c-9b53-e492aaf5c95f.png)

**Comment by charliecruzan-stripe** [comment]
Hi! Expo's latest SDK (47) supports v0.19.0. Can you install stripe with `expo install @stripe/stripe-react-native` and retry?

**Comment by ryanwaite28** [comment]
Hi, will retry.

**Comment by ryanwaite28** [comment]
Just tried and it did not work. Here is what my package.json looks like:

```json
{
  "name": "app",
  "version": "1.0.0",
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "NODE_ENV=production expo start",
    "start-local": "NODE_ENV=local expo start",
    "start-dev": "NODE_ENV=development expo start",
    "android": "NODE_ENV=local expo start --android",
    "ios": "NODE_ENV=local expo start --ios",
    "web": "NODE_ENV=local expo start --web",
    "build-pre-release": "NODE_ENV=development eas update --branch=preview"
  },
  "dependencies": {
    "@googlemaps/google-maps-services-js": "^3.3.16",
    "@react-native-async-storage/async-storage": "^1.17.10",
    "@react-native-picker/picker": "2.4.8",
    "@react-navigation/bottom-tabs": "^6.4.0",
    "@react-navigation/drawer": "^6.5.0",
    "@react-navigation/material-top-tabs": "^6.2.4",
    "@react-navigation/native": "^6.0.13",
    "@react-navigation/native-stack": "^6.9.0",
    "@stripe/stripe-react-native": "0.19.0",
    "axios": "^0.27.2",
    "buffer": "^6.0.3",
    "dotenv": "^16.0.3",
    "eslint-plugin-react-hooks": "^4.6.0",
    "expo": "^47.0.8",
    "expo-device": "~5.0.0",
    "expo-image-picker": "~14.0.2",
    "expo-linking": "~3.2.3",
    "expo-location": "~15.0.1",
    "expo-notifications": "~0.17.0",
    "expo-status-bar": "~1.4.2",
    "expo-web-browser": "~12.0.0",
    "install": "^0.13.0",
    "moment": "^2.29.4",
    "npm": "^8.19.2",
    "react": "18.1.0",
    "react-dom": "18.1.0",
    "react-native": "0.70.5",
    "react-native-app-link": "^1.0.1",
    "react-native-encrypted-storage": "^4.0.2",
    "react-native-gesture-handler": "~2.8.0",
    "react-native-google-places-autocomplete": "^2.4.1",
    "react-native-image-picker": "^4.10.0",
    "react-native-keyboard-aware-scroll-view": "^0.9.5",
    "react-native-maps": "1.3.2",
    "react-native-pager-view": "6.0.1",
    "react-native-reanimated": "~2.12.0",
    "react-native-safe-area-context": "4.4.1",
    "react-native-screens": "~3.18.0",
    "react-native-signature-canvas": "^4.4.1",
    "react-native-signature-capture": "^0.4.12",
    "react-native-tab-view": "^3.1.1",
    "react-native-web": "~0.18.7",
    "rxjs": "^7.5.6",
    "socket.io-client": "^4.5.1",
    "thenby": "^1.3.4",
    "uri-scheme": "^1.0.120",
    "expo-secure-store": "~12.0.0",
    "expo-updates": "~0.15.6",
    "@expo/webpack-config": "^0.17.0"
  },
  "devDependencies": {
    "@babel/core": "^7.12.9",
    "@types/react": "~18.0.14",
    "@types/react-native": "~0.69.1",
    "typescript": "~4.3.5"
  },
  "private": true
}

```

**Comment by ryanwaite28** [comment]
I found the issue. Basically, no property in the `billingDetails` object should be null/undefined:
```typescript
const createCardParams: CreateParams = {
        paymentMethodType: 'Card',
        paymentMethodData: {
          billingDetails: {
            name: getUserFullName(you),
            email: you.email,
            phone: you.phone || '',
          },
        }
      };
```

If you don't have a value for it, just use an empty string, -1, etc. Just something other than null/undefined.

**Comment by charliecruzan-stripe** [comment]
Hm, I'm not able to reproduce that @ryanwaite28

**Comment by woohoou** [comment]
+1 It only happens in release environments

**Comment by woohoou** [comment]
I confirm @ryanwaite28 solution. Thank you!

**Comment by fernando-orchata** [comment]
I was stuck in the same issue, thanks @ryanwaite28 !! Your solution worked!

**Comment by charliecruzan-stripe** [comment]
Fix was released in [v0.23.0](https://github.com/stripe/stripe-react-native/releases/tag/v0.23.0)
